Jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
     // AND someone confirms this stage within 5 minutes
     stage('Conditional publish to RubyGems and Dockerhub') {
       // Only run this stage when triggered by a tag
-      when { tag "v*" }
+      //when { tag "v*" }
```

Comment:
Not sure if this should stay commented?

Reply:
yup. uncommented this

Reply:
Ah, @neil-k-zero , just remembered it was commented out because otherwise I can't really test it, unless I tag a new version of the gem